code/modules/jobs/job_types/gimmick/departmentcaptain.dm

```diff
@@ -0,0 +1,128 @@
+GLOBAL_LIST_INIT(captain_departments, list(
+	"Control",
+	"Information",
+	"Training",
+	"Safety",
+	"Welfare",
+	"Discipline",
+	))
+
+
+/datum/job/departmentcaptain
+	title = "Department Captain"
+	department_head = list("Manager")
+	faction = "Station"
+	total_positions = 6
+	spawn_positions = 6
+	supervisors = "the manager"
+	selection_color = "#ccaaaa"
+	exp_requirements = 240
+	exp_type = EXP_TYPE_CREW
+	exp_type_department = EXP_TYPE_SECURITY
+
+	outfit = /datum/outfit/job/departmentcaptain
+	display_order = JOB_DISPLAY_ORDER_SCIENTIST
+
+	access = list()
+	minimal_access = list()
+
+	allow_bureaucratic_error = FALSE
+
+	roundstart_attributes = list(
+								FORTITUDE_ATTRIBUTE = 20,
+								PRUDENCE_ATTRIBUTE = 20,
+								TEMPERANCE_ATTRIBUTE = 20,
+								JUSTICE_ATTRIBUTE = 20
+								)
+	maptype = "wonderlabs"
+	var/normal_attribute_level = 20 // Scales with round time
+
+/datum/job/departmentcaptain/after_spawn(mob/living/carbon/human/H, mob/M, latejoin = FALSE)
+	var/set_attribute = normal_attribute_level
+	if(world.time >= 75 MINUTES) // Full facility expected
+		set_attribute *= 4
+	else if(world.time >= 60 MINUTES) // More than one ALEPH
+		set_attribute *= 3
+	else if(world.time >= 45 MINUTES) // Wowzer, an ALEPH?
+		set_attribute *= 2.5
+	else if(world.time >= 30 MINUTES) // Expecting WAW
+		set_attribute *= 2
+	else if(world.time >= 15 MINUTES) // Usual time for HEs
+		set_attribute *= 1.5
```

Comment:
We should really make this a switch statement or something later. More of a comment not a request.

Reply:
Will do this eventually, and also refactor.